sound/soc/sof/sof-priv.h

```diff
@@ -430,16 +420,8 @@ struct snd_sof_dev {
 	int ipc_timeout;
 	int boot_timeout;
 
-	/* DMA for Trace */
-	struct snd_dma_buffer dmatb;
-	struct snd_dma_buffer dmatp;
-	int dma_trace_pages;
-	wait_queue_head_t trace_sleep;
-	u32 host_offset;
-	bool dtrace_is_supported; /* set with Kconfig or module parameter */
-	bool dtrace_is_enabled;
-	bool dtrace_error;
-	bool dtrace_draining;
+	/* dtrace client is loaded */
+	bool dtrace_is_loaded;
```

Comment:
"loaded" isn't very clear to me. Can we reuse "dtrace_is_enabled?"

Reply:
I'll use `dtrace_is_available`